### DISCUSSION TITLE
"nillion-devnet" command is not working

### DISCUSSION THREAD

**Discussion by arnavbee** [NillionNetwork/discussions#134] [Q&A]
I'm following this tutorial: https://github.com/NillionNetwork/awesome-nillion/issues/2

after running the "nillion-devnet" command in another terminal window,

it is showing 

> command not found: nillion-devnet

**Comment by wwwehr** [comment]
as part of the init, the dir of the SDK bins is added to your PATH environment var. if you open a new terminal does it give you the same error? which system are you running on? (eg. linux/mac)

**Reply by arnavbee** [reply]
I'm running on Mac
